### PR TITLE
Remove data, metric and common to neural_compressor

### PR DESCRIPTION
**Pull request by changwangss**
## Type of Change

refine class name
remove data,metric,common to neural_compressor

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: neural_compressor/adaptor/mxnet_utils/util.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 from enum import Enum
 from tempfile import TemporaryDirectory
 from neural_compressor.utils.utility import LazyImport
-from neural_compressor.model.model import MXNetModel as NCModel
+from neural_compressor.model.mxnet_model import MXNetModel as NCModel
```
Comment (ftian1) [inline neural_compressor/adaptor/mxnet_utils/util.py:27]: we need add MXNetModel and others to __init__.py for top namespace

Comment (changwangss) [reply neural_compressor/adaptor/mxnet_utils/util.py:27]: will improve it in bug fix period.

**Review by ftian1 — APPROVED** [review]

**Pull request merged**